tests/suites/0_stateless/02_0006_function_argmin_argmax.result

```diff
@@ -0,0 +1,46 @@
+--------------
+SELECT sum(number) AS salary, number%3 AS user FROM numbers_mt(10000) GROUP BY user ORDER BY salary ASC;
```

Comment:
Good job, but wanna know why https://github.com/datafuselabs/datafuse/issues/657 not works

Reply:
Look like that the logic we use to process with priorities is flawed.
By the way, we didn't fix this problem in #469, we didn't process with priority in #469 (maybe in the next patch)

reference : https://github.com/datafuselabs/datafuse/pull/469/files#diff-245f1e3522aeff587d228129a562a72c9d4754d17cfe60d9e627aa7889920121L107

Reply:
Cc @jyizheng